pkg/envoy/bootstrap/envoy_config_health_probes.go

```diff
@@ -33,25 +33,25 @@ const (
 	startupListener   = "startup_listener"
 )
 
-func getLivenessCluster(originalProbe *models.HealthProbe) *xds_cluster.Cluster {
+func getLivenessCluster(clusterName string, originalProbe *models.HealthProbe) *xds_cluster.Cluster {
```

Comment:
nit: change change get to generate/build

Reply:
done!